test/resource_reviews.test.js

```diff
@@ -0,0 +1,99 @@
+import Listings from '../src/resources/listings.js'
+import Purchase from '../src/resources/purchases.js'
+import Reviews from '../src/resources/reviews'
+import IpfsService from '../src/services/ipfs-service'
+import { expect } from 'chai'
+import Web3 from 'web3'
+import asAccount from './helpers/as-account'
+import contractServiceHelper from './helpers/contract-service-helper'
+
+const samplePrice = 1000000000000000
+const listingData = {
+  priceWei: samplePrice,
+  unitsAvailable: 1,
+  listingType: 'unit',
+  name: 'Foo Bar'
+}
+const purchaseData = {
+  priceWei: samplePrice,
+  units: 1,
+  purchaseType: 'unit'
+}
+const reviewData_1 = {
+  rating: 2,
+  description: 'lol'
+}
+const reviewData_2 = {
+  rating: 5,
+  description: 'omg wat'
+}
+
+describe('Review Resource', function() {
+  this.timeout(5000) // default is 2000
+
+  let reviews,
+    accounts,
+    buyer,
+    contractService,
+    listings,
+    listingIndex,
+    purchases,
+    purchaseIndex
+
+  beforeEach(async () => {
+    const ipfsService = new IpfsService({
+      ipfsDomain: '127.0.0.1',
+      ipfsApiPort: '5002',
+      ipfsGatewayPort: '8080',
+      ipfsGatewayProtocol: 'http'
+    })
+    const provider = new Web3.providers.HttpProvider('http://localhost:8545')
```

Comment:
not specific to this PR, but we should consider using ephemeral ports when binding services during tests. this helps avoid port conflicts between a local development blockchain and test blockchains

Reply:
note that this is not for this PR... just noting